lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -131,19 +133,22 @@
 
                     this.mainView = this.tabbedView = this.createViewWithHeader({
                         title: gettext("Teams"),
-                        description: gettext("See all teams in your course, organized by topic. Join a team to collaborate with other learners who are interested in the same topic as you are."),
+                        description: gettext("See all teams in your course, organized by topic. Join a team to collaborate with other learners who are interested in the same topic as you are."),  // jshint ignore:line
                         mainView: new TeamsTabbedView({
                             tabs: [{
                                 title: gettext('My Team'),
                                 url: 'my-teams',
                                 view: this.myTeamsView
                             }, {
-                                title: interpolate(
+                                title: HtmlUtils.interpolateHtml(
```

Comment:
Now I see how title could be plain text or HTML.  I think the name of this is fine as-is.